src/components/Accordion/_accordion.scss

```diff
@@ -98,7 +98,7 @@
     padding-right: var(--space-lg);
   }
 
-  .cc-info-box & {
+  .c-info-box & {
```

Comment:
@creido-welly, do you think it would be better to move that (and below) style to corporate-react?

Reply:
Yes move this style to infobox stylesheet